### PR TITLE
rename XLS-20 fields

### PR DESCRIPTION
**Pull request by khancode**
## High Level Overview of Change

Rename XLS-20 fields to match rippled v1.9.0. [Github Issue](https://github.com/XRPLF/xrpl.js/issues/1965)

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: packages/xrpl/src/models/methods/nftBuyOffers.ts [hunk 1]
```diff
@@ -30,6 +30,6 @@ export interface NFTBuyOffersResponse extends BaseResponse {
     /**
      * The token ID of the NFToken to which these offers pertain.
      */
-    tokenid: string
+    nftokenid: string
```
Comment (ledhed2222) [inline packages/xrpl/src/models/methods/nftBuyOffers.ts:33]: this is `nft_id` not `nftokenid`

File: packages/xrpl/src/models/methods/nftBuyOffers.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export interface NFTBuyOffersRequest extends BaseRequest {
   /**
    * The unique identifier of an NFToken. The request returns buy offers for this NFToken.
    */
-  tokenid: string
+  nftokenid: string
```
Comment (ledhed2222) [inline packages/xrpl/src/models/methods/nftBuyOffers.ts:16]: this is nft_id not nftokenid

File: packages/xrpl/src/models/methods/nftSellOffers.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ export interface NFTSellOffersRequest extends BaseRequest {
   /**
    * The unique identifier of an NFToken. The request returns sell offers for this NFToken.
    */
-  tokenid: string
+  nftokenid: string
```
Comment (ledhed2222) [inline packages/xrpl/src/models/methods/nftSellOffers.ts:16]: this is nft_id not nftokenid

File: packages/xrpl/src/models/methods/nftSellOffers.ts [hunk 4]
```diff
@@ -30,6 +30,6 @@ export interface NFTSellOffersResponse extends BaseResponse {
     /**
      * The token ID of the NFToken to which these offers pertain.
      */
-    tokenid: string
+    nftokenid: string
```
Comment (ledhed2222) [inline packages/xrpl/src/models/methods/nftSellOffers.ts:33]: this is nft_id not nftokenid

**Comment by JST5000** [comment]
We should make a separate PR to update xrpl.js models to reflect the new changes as well. (It seems like too many changes to add it to this PR)

**Review by ckniffen — APPROVED** [review]

**Review by ckniffen — APPROVED** [review]

File: packages/ripple-binary-codec/HISTORY.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
 # ripple-binary-codec Release History
 
+## 1.3.3 (2022-04-18)
+- Updated NFT definitions to match 1.9.0's breaking naming changes
```
Comment (ledhed2222) [inline packages/ripple-binary-codec/HISTORY.md:4]: i might make this 1.4.0

**Review by ledhed2222 — APPROVED** [review]
i might make this 1.4.0

**Pull request merged**